### ISSUE TITLE
Network infinite scroll skips three days worth of posts

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#4835]
### Expected behavior

The network page in post order shows all posts in decreasing order of post date.

### Actual behavior

At some point, the listing jumps from "1 hour ago" to "3 days ago" and keeps showing older posts onwards. I can't access posts created in the meantime through the network page, however they still are accessible through notifications.

### Steps to reproduce the problem
- Have more than 3 days post history
- Select Post Order in the network page
- Enable infinite scrolling
- Scroll down the network page, check the dates out

### Friendica version you encountered the problem

develop

### Friendica source (git, zip)

git

### PHP version

7.0

### SQL version

MariaDB 10.14

![screen shot 2018-04-13 at 1 24 07 pm](https://user-images.githubusercontent.com/925415/38749014-1c9aa12e-3f1e-11e8-9a6c-bc03585490f6.png)

**Comment by annando** [comment]
Question is, after which field you ordered your posts. If it was after the last comment, then a "like" is changing the "comment" field.

**Comment by MrPetovan** [comment]
It's showing the "Post Order", I didn't change this setting since I first settled on it about a year ago.

**Comment by MrPetovan** [comment]
My network page just skipped 22 hours worth of content again. I read and interacted with many posts this morning but nothing shows up until 22 hours ago.

**Comment by MrPetovan** [comment]
My network stream just skipped 1 week worth of posts after I replied a couple times on a 1 week old Diaspora post. My network stream then resumes from the week-old post date.

**Comment by MrPetovan** [comment]
My network stream now shows an insane list after having been open for a while:
- 14 minutes ago
- 21 minutes ago (2 posts)
- 22 minutes ago
- 23 minutes ago (2 posts)
- 26 minutes ago
- 17 hours ago
- 23 hours ago
- 2 days ago
- 3 days ago (5 posts)
- 4 days ago (2 posts)
- 5 days ago (3 posts)
- 2 weeks ago
- 23 seconds ago
- 40 minutes ago
- 4 minutes ago
- 8 minutes ago
- 11 minutes ago

After refresh all older posts are gone and ordering is restored.

**Comment by annando** [comment]
This could be something with the update section of the network page.

**Comment by MrPetovan** [comment]
Of course, I’m still surprised to see the 2 weeks old post. Maybe it still operates with “commented” even with the “posted” mode on the main page?

**Comment by MrPetovan** [comment]
Just happened again, with "Posted last" tab post times are:
- 3 minutes ago
- 17 minutes ago
- 19 minutes ago
- 21 minutes ago
- 23 minutes ago
- 30 minutes ago
- 32 minutes ago
- 36 minutes ago
- 36 minutes ago
- 38 minutes ago
- 41 minutes ago
- 44 minutes ago
- 44 minutes ago
- 44 minutes ago
- 47 minutes ago
- 6 days ago (I replied 6 minutes ago on this one)
- 6 days ago
- ...

**Comment by AlfredSK** [comment]
Yep. Confirmed. I have a jump from '6 hours ago' to '6 days ago'.

**Comment by tobiasd** [comment]
@hoergen [reported](https://forum.friendi.ca/display/2196d4ee-545c-8f43-b744-386307623032) that he had this problem without the infinite-scroll active.

**Comment by AlfredSK** [comment]
Örgs. There it is again 2 hours ago -> 5 days ago.

Reloading of the network page doesn't help. BUT...
When you delete the first old item after the leap in time and reload the page afterwards everything is OK again.

**Comment by MrPetovan** [comment]
Oh, interesting, thanks for the courageous move.
